### PR TITLE
Primeng added components

### PR DESCRIPTION
**Pull request by YuliiaMilian**

### CODE REVIEW

File: src/app/feedback-form/feedback-form.component.html [hunk 1]
```diff
@@ -1,54 +1,75 @@
...
       <header class="contact-form__header">
         <h3 class="contact-form__title">Get in Touch with Us!</h3>
         <p class="contact-form__subtitle">We'd love to hear from you. Please fill out the form below.</p>
       </header>
 
       <div class="contact-form__form-group">
-        <label class="contact-form__label" for="name">Your Name:</label>
-        <input placeholder="Enter your name" class="contact-form__input" type="text" id="name" name="name" required />
+        <p-floatLabel>
+          <input class="contact-form__input" type="text" id="name" formControlName="name" pInputTextarea />
```
Comment (Khousheish) [inline src/app/feedback-form/feedback-form.component.html:11]: this needs to be text not textarea

File: src/app/feedback-form/feedback-form.component.html [hunk 2]
```diff
@@ -1,54 +1,75 @@
...
-        <input
-          placeholder="Enter your email"
-          class="contact-form__input"
-          type="email"
-          id="email"
-          name="email"
-          required
-        />
+        <p-floatLabel>
+          <input class="contact-form__input" type="email" id="email" formControlName="email" pInputTextarea />
```
Comment (Khousheish) [inline src/app/feedback-form/feedback-form.component.html:23]: same here

File: src/app/feedback-form/feedback-form.component.html [hunk 3]
```diff
@@ -1,54 +1,75 @@
...
-        <input
-          placeholder="Subject of your message"
-          class="contact-form__input"
-          type="text"
-          id="subject"
-          name="subject"
-          required
-        />
+        <p-floatLabel>
+          <input class="contact-form__input" type="text" id="subject" formControlName="subject" pInputTextarea />
```
Comment (Khousheish) [inline src/app/feedback-form/feedback-form.component.html:39]: and here

File: src/app/feedback-form/feedback-form.component.html (hunk 1 above)
Comment (YuliiaMilian) [reply src/app/feedback-form/feedback-form.component.html:11]: @Khousheish When I use pInputText, the label in p-floatLabel overlaps the text after clicking outside the field
Screenshot 2024-10-30 at 13 02 44
=

Comment (YuliiaMilian) [reply src/app/feedback-form/feedback-form.component.html:11]: In field 'Enter name" I use approach with pInputTextarea

**Pull request merged**